### DISCUSSION TITLE
Shouldn't the accordion consist of a details element and a summary element?

### DISCUSSION THREAD

**Discussion by k98a73** [mantinedev/mantine#3164] [Feedback]
A word search on the site does not find the word in the accordion.

That is because the accordion is composed of `div` and `button` elements.
Composing the accordion with a `details` element and a `summary` element would solve this problem.

If you are willing to accept this fix, I would be happy to take charge of the fix.

**Comment by rtivital** [comment]
There is no reason to use these elements for building accordion. Component is accessible already.

**Comment by k98a73** [comment]
@rtivital 
Thanks for answering.
When I do a site search with the accordion closed, the accordion does not open even if the corresponding word is found.

**Reply by rtivital** [reply]
You will not be able to build the same component with these elements

**Reply by k98a73** [reply]
I understand.

**Comment by lightsound** [comment]
The details and summary elements are more accessible.
https://user-images.githubusercontent.com/8220973/207038566-cb6ce621-e14a-4c4e-b4e1-7a69924ee197.mp4

**Reply by rtivital** [reply]
You will not be able to build the same component with these elements
